ldm/generate.py

```diff
@@ -220,8 +221,14 @@ def __init__(
                 from diffusers.pipelines.stable_diffusion.safety_checker import StableDiffusionSafetyChecker
                 from transformers import AutoFeatureExtractor
                 safety_model_id = "CompVis/stable-diffusion-safety-checker"
-                self.safety_checker = StableDiffusionSafetyChecker.from_pretrained(safety_model_id, local_files_only=True)
-                self.safety_feature_extractor = AutoFeatureExtractor.from_pretrained(safety_model_id, local_files_only=True)
+                self.safety_checker = StableDiffusionSafetyChecker.from_pretrained(safety_model_id,
+                                                                                   local_files_only=True,
+                                                                                   cache_dir=os.path.join(Globals.root,'models',safety_model_id)
```

Comment:
Would be a good idea to define a variable for this path join so that if someone changes it in the future they don't accidentally change only one of them.

Reply:
Good point. Will fix.

Reply:
Done.

Reply:
This is now fixed and tested.